concepts/patterns/introduction.md

```diff
@@ -59,17 +59,19 @@ AWK gives you a whole programming language without sacrificing succinctness.
 
 We'll get more into GNU AWK's regular expression flavour in a further concept.
 
-## Arithmetic expressions
+## Expressions
 
-Arithmetic expressions can be used as patterns.
+AWK expressions (arithmetic or logical or ...) can be used as patterns.
 
 To extract all users with UID 1000 or above:
 
 ```sh
 awk -F: '$3 >= 1000' /etc/passwd
 ```
 
-Recall that zero is a false value and all other numbers are true.
+Recall that zero is a false value and all other numbers are true;
+and the empty string is a false value and all other strings are true.
```

Comment:
```suggestion
Recall that zero is a false value and all other numbers are true.
The empty string is a false value and all other strings are true.
```

Reply:
I'm asking people to recall all of that. Two sentences doesn't feel right.